samples/go/csv/csv-import/perf_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/attic-labs/noms/go/datas"
+	"github.com/attic-labs/noms/go/dataset"
+	"github.com/attic-labs/noms/go/hash"
+	"github.com/attic-labs/noms/go/perf/suite"
+	"github.com/attic-labs/noms/go/types"
+	humanize "github.com/dustin/go-humanize"
+)
+
+// CSV perf suites require the testdata directory to be checked out at $GOPATH/src/github.com/attic-labs/testdata (i.e. ../testdata relative to the noms directory).
+
+// TODO: Add ny-vehicle-registrations test when CSV importing is faster (testdata/ny-vehicle-registrations/20150218.*).
+
+type perfSuite struct {
+	suite.PerfSuite
+	server      *datas.RemoteDatabaseServer
+	sfcBlobHash hash.Hash
+}
+
+func (s *perfSuite) Test01ImportSfCrimeBlobFromTestdata() {
+	assert := s.NewAssert()
+
+	var raw []io.Reader
+
+	s.Pause(func() {
+		// The raw data is split into a bunch of files foo.a, foo.b, etc.
+		glob, err := filepath.Glob(path.Join(s.AtticLabs, "testdata", "sf-crime", "2016-07-28.*"))
+		assert.NoError(err)
+
+		raw = make([]io.Reader, len(glob))
```

Comment:
Why not just make these io.ReadClosers? then you don't need to type assert below